### PR TITLE
docs: Updating README for a QuickStart

### PR DESCRIPTION
**Pull request by antoniomtz**
## PR Checklist

- [ x] Added label to the Pull Request for easier discoverability and search
- [x ] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [ ] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [x ] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## Issue this PR will close

close: #**issue_number**

### CODE REVIEW

**Review by brian-intel — APPROVED** [review]
LGTM

**Review by NeethuES-intel — APPROVED** [review]
LGTM

**Review by vli11 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,6 +2,61 @@
...
+./download_sample_videos.sh
+```
+
+3. Run camera simulator:
+
+(Go back to the root directory)
+
+```
+cd ..
+./camera-simulator/camera-simulator.sh
```
Comment (jim-wang-intel) [inline README.md:42]: i think there is also a make target for running camera-simulator: `make run-camera-simulator`.

File: README.md [hunk 2]
```diff
@@ -2,6 +2,61 @@
...
+```
+
+4. Run object detection:
+
+(This first command will take some time while it downloads the models)
```
Comment (jim-wang-intel) [inline README.md:47]:
```suggestion
(This first command will take some time while it downloads the models for the first time)
```

**Pull request merged**